ast/ast.h

```diff
@@ -0,0 +1,47 @@
+#ifndef _AST_H_
+#define _AST_H_
+
+typedef enum {
+	node_type_command,
+	node_type_literal_string
+} e_node_type;
+
+typedef enum {
+	value_type_string,
+	value_type_integer,
+	value_type_undefined
+} e_value_type;
+
+typedef union {
```

Comment:
I removed it to simplify things (it wasn't being used). We can put it back when we need to handle types other than strings.